docs/user_guide/5_setup_methods/index.rst

```diff
@@ -0,0 +1,19 @@
+Setup Methods
```

Comment:
Maybe Technical Description is better than Setup methods. If you think Technical description of the setup methods is too long
```suggestion
Technical description
```